site/src/index.css

```diff
@@ -31,6 +31,10 @@
 		--border: 240 5.9% 90%;
 		--input: 240 5.9% 90%;
 		--ring: 240 10% 3.9%;
+		--muted: 240, 6%, 90%;
```

Comment:
We dont have tokens muted, popover, and accent in our design system. It should be possible to map these to the colors that exist there, https://www.figma.com/design/WfqIgsTFXN2BscBSSyXWF8/Coder-kit?node-id=1-2&t=ZPGincsj3IkIlTun-1

Reply:
I see, I’ll do that. I’m just wondering how painful it is to map each one of these every time we add a new component 🥲.

Reply:
This does not happen for all components so overall its not been too bad so far